### DISCUSSION TITLE
My profile and uploaded styles are gone - bug?

### DISCUSSION THREAD

**Discussion by GoldenMeadows** [userstyles-world/userstyles.world#328] [Q&A]
I just went to upload a new style for a browser game I play and it gave me a "duplicate name" error. Okay, fair enough. I've tried changing the name 3 different times and it refused to upload, so I tried checking my profile to see if it was up already.

My profile is gone.

Linking to my page now gives a "user not found" error. My old styles? Also gone. The links give "not found" errors. What happened here? If it's a bug I'll gladly report it, but I want to make sure it's not user error somehow first.

[Link to my profile](https://userstyles.world/user/goldenmeadows)
[Link to an old style](https://userstyles.world/style/8893/ffh-navy-gold)

[Link to screenshot of my profile](https://i.postimg.cc/tJvDwV59/image.png)

Any help that can be given is greatly appreciated.

Edit: Forgot to add an image but it still says I'm logged in. I'm afraid to log out just in case everything I had goes poof permanently. 

[Screenshot](https://i.postimg.cc/QthFG44q/image.png)

2nd edit: it works now? Not sure what kind of hiccup happened there but the screenshots above show something definitely happened. Probably a server error or something I'd assume?

**Comment by astyled** [comment]
Likely it was a temporal issue with the database. I had the same issue a few days ago, but it only lasted a few seconds.

Thank you for documenting this! I suppose your issue is resolved now?
